corehq/apps/geospatial/tests/test_index_geolocation_case_properties.py

```diff
@@ -0,0 +1,76 @@
+from django.test import TestCase
+
+from casexml.apps.case.mock import CaseFactory
+
+from corehq.apps.es import CaseSearchES
+from corehq.apps.es.case_search import case_search_adapter
+from corehq.apps.es.tests.utils import es_test
+from corehq.apps.geospatial.management.commands.index_geolocation_case_properties import (
+    _case_query,
+    index_case_docs,
+)
+from corehq.apps.geospatial.models import GeoConfig
+
+
+@es_test(requires=[case_search_adapter], setup_class=True)
+class TestGetFormCases(TestCase):
+    domain = 'foobar'
+    primary_case_type = 'primary'
+    secondary_case_type = 'secondary'
+    gps_prop_name = 'gps_coordinates'
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        factory = CaseFactory(cls.domain)
+        case1 = factory.create_case(
+            case_name='foo',
+            case_type=cls.primary_case_type,
+            update={cls.gps_prop_name: '1.1 2.2'},
+        )
+        case2 = factory.create_case(
```

Comment:
Can we also add a case without `gps_prop_name` properties?

Reply:
Addressed in [cee0895](https://github.com/dimagi/commcare-hq/pull/35090/commits/cee089581f705ba22bacc7348a6c4339a1259b48).